### PR TITLE
feat: update tribe landing page

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

**Landing Page**
- updated the landing page based on new design
- compressed svgs

TODO: wire in redirection to typeform / mail when user selects the plan 
(pending @vs77bb  / @PixelantDesign )

**User Directory**
- toggle filter visibility if the persona is tribes
- toggle banner for tribes
- refactor template 
- updated the layout for tribe card  
- custom url for only tribes (/tribes/explore) [ filters are hidden by default ]

Video : https://share.vidyard.com/watch/68GZm6kMa9KPAETR84mS1Q?

TODO: add in custom filters for tribes page
(pending @PixelantDesign )

screencapture-localhost-8000-tribes-2020-02-17-13_24_48

### CODE REVIEW

File: app/assets/v2/css/retail/tribes_landing.css [hunk 1]
```diff
@@ -7,28 +7,19 @@
...
-}
-
-.companies img {
-  height: 3.5rem;
-  transition: transform .25s;
-}
-
-.companies img:last-child {
-  margin-bottom: 0!important;
+.gc-blue-text {
```
Comment (octavioamu) [inline app/assets/v2/css/retail/tribes_landing.css]: this belong here or to gitcoin.css?

File: app/assets/v2/css/users.css [hunk 2]
```diff
@@ -22,6 +22,10 @@
   }
 }
 
+.users .dropdown-menu {
```
Comment (octavioamu) [inline app/assets/v2/css/users.css]: this need to be removed, will affect the nav dropdowns and every dropdown on this page, that is a bootstrap class and that component is used inside others components.

File: app/assets/v2/css/retail/tribes_landing.css [hunk 3]
```diff
@@ -47,53 +38,102 @@ img {
...
+  background: white;
 }
 
 .testimonial-section .testimonial-photo-container {
   top: -2rem;
   left: 0;
   right: 0;
+}
+
+#pro,
```
Comment (octavioamu) [inline app/assets/v2/css/retail/tribes_landing.css]: why not just 
```
#pro {
    color:white
    background-color: #170640;
}  
```  
or with a class always is better.

File: app/assets/v2/css/users.css (hunk 2 above)
Comment (thelostone-mc) [reply app/assets/v2/css/users.css]: updated

File: app/assets/v2/css/retail/tribes_landing.css (hunk 3 above)
Comment (thelostone-mc) [reply app/assets/v2/css/retail/tribes_landing.css]: ^ Cause the h2 + others have have a colo black assigned to them in plans.css 
This is just for users and hence the reason for writing it up

**Pull request merged**